resources/views/auth/form.blade.php

```diff
@@ -11,6 +11,11 @@
     {!! Form::text('email', null, ['placeholder' => 'you@example.com']) !!}
 @endif
 
+@if(should_collect_international_phone())
+    {!! Form::label('phone', 'Cell Number (optional)') !!}
+    {!! Form::text('phone', null, ['placeholder' => '555-555-5555']) !!}
```

Comment:
should this be an `||` with the statement below? or do we want to make them feel welcome, and put things in the placeholder that fit their phone number styleeee?